modules/openapi-generator/src/main/resources/swift5/Package.swift.mustache

```diff
@@ -1,12 +1,12 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.1
 
 import PackageDescription
 
 let package = Package(
     name: "{{projectName}}",
     platforms: [
         .iOS(.v9),
-        .macOS(.v10_11),
+        .macOS(.v10_15),
```

Comment:
I think here we should put an if uses Vapor

Reply:
Agree.